docs/SUMMARY.md

```diff
@@ -125,19 +117,25 @@
     * [Performance](guides/performance.md)
     * [Concurrency](guides/concurrency.md)
     * [Testing](guides/testing.md)
+    * [The shards command](the_shards_command/README.md)
     * [Writing Shards](guides/writing_shards.md)
         * [Hosting on GitHub](guides/hosting/github.md)
         * [Hosting on GitLab](guides/hosting/gitlab.md)
     * [Continuous Integration](guides/ci/README.md)
         * [GitHub Actions](guides/ci/gh-actions.md)
         * [CircleCI](guides/ci/circleci.md)
+    * [Using the compiler](using_the_compiler/README.md)
```

Comment:
Is there any particular reasoning to these insert positions? It seems very random.
Not that the order of existing items is very sophisticated, but when adding stuff we should at least try to make it fit in a sensible way.
The shards manual should  come after the compiler, for example. And we could consider to place them in a new sub section called "Manuals".